masterlist.yaml

```
@@ -28190,3 +28190,49 @@ plugins:
         crc: 0x9E8F3CA5
         util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 17
+        
+  - name: 'UnforgivingDevices - DD Patch.esp'
+    url:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'unforgiving Devices DD Patch'
+    after:
+      - 'Devious Devices SE patch.esp'
+      - 'Devious Devices - Assets.esm'
+      - 'Devious Devices - Expansion.esm'
+      - 'Devious Devices - Integration.esm'
+      - 'Devious Devices For Him.esp'
+      - 'Devious Devices - BRRF.esp'
+      - 'Devious Devices - Lore Patch.esp'
+    clean:
+      - crc: 0xD32A2D77
+        util: 'SSEEdit v4.0.4'
+  - name: 'UnforgivingDevices - LBS Patch.esp'
+    url:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'unforgiving Devices Lauras Bondage Shop Patch'
+    after:
+      - 'UnforgivingDevices - DD Patch.esp'
+      - 'Laura's Bondage Shop.esp'
```

Apostrophes need to be escaped (or this plugin should be removed if it's redundant):

```suggestion
      - 'Laura''s Bondage Shop.esp'
```